src/include/daos_pool.h

```diff
@@ -183,6 +183,10 @@ typedef struct {
 	struct daos_pool_space		pi_space;
 	/** rebuild status */
 	struct daos_rebuild_status	pi_rebuild_st;
+	/** Current pool global version */
```

Comment:
@mchaarawi Any idea for this?

Reply:
i don't believe that this would be a major version change. we are not removing any members of that struct and we are not adding or replacing any parameters in the middle of that struct. so it won't require any apps to be rebuilt.

Reply:
It's an ABI change with a caller-exposed structure size change. Wouldn't that constitute a major API change then (and require updating the daos RPM packages / version)?

As an experiment (hopefully not a flawed one) I tried with a source build and the daos_test binary (rather than testing with middleware and RPMs). First, building from source, from the PR's basis on master, commit 5841ad6. And then building this PR. Running the old daos_test binary from the first build (originally used with libdaos.so.2 -> libdaos.so-2.3.0)  with the new daos build (now libdaos.so.2 -> libdaos.so-2.4.0) results in a failure
[  FAILED  ] POOL10: pool create with properties and query